deepmd/pd/model/descriptor/repformer_layer_old_impl.py

```
@@ -0,0 +1,752 @@
+    def _cal_h2g2(
+        self,
+        g2: paddle.Tensor,
+        h2: paddle.Tensor,
+        nlist_mask: paddle.Tensor,
+        sw: paddle.Tensor,
+    ) -> paddle.Tensor:
+        # g2:  nf x nloc x nnei x ng2
+        # h2:  nf x nloc x nnei x 3
+        # msk: nf x nloc x nnei
+        nb, nloc, nnei, _ = g2.shape
+        ng2 = g2.shape[-1]
```

Ruff

Local variable `ng2` is assigned to but never used

Remove assignment to unused variable `ng2`

(F841)